### ISSUE TITLE
TypeScript error if imported package contains `<reference types="node" />`

### ISSUE THREAD

**Issue by yuanqing** [yuanqing/create-figma-plugin#71]
```
$ yarn run build
yarn run v1.22.17
warning package.json: No license field
$ build-figma-plugin --typecheck --minify
info Type checking...
error TypeScript error

    node_modules/@figma/plugin-typings/index.d.ts:10:9 - error TS2649: Cannot augment module 'console' with value exports because it resolves to a non-module entity.

    10   const console: Console
               ~~~~~~~

error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```

**Comment by yuanqing** [comment]
Fixed as of `1.7.2`

**Comment by shcallaway** [comment]
What exactly was the fix here?
